### ISSUE TITLE
 ERROR  Cannot read properties of undefined (reading 'options')

### ISSUE THREAD

**Issue by mklueh** [nuxt-community/eslint-module#78]
I'm suddenly running into an issue with this plugin

```
# This file contains the result of Yarn building a package (my-app-8b06de@workspace:apps/my-app)
# Script name: postinstall

Nuxi 3.0.0

 ERROR  Cannot read properties of undefined (reading 'options')

  at nuxtModule (/home/user/workspace/my-project/node_modules/@nuxtjs/eslint-module/dist/module.js:25:13)
  at installModule (/home/user/workspace/my-project/node_modules/@nuxt/kit/dist/index.mjs:416:9)
  at async initNuxt (/home/user/workspace/my-project/node_modules/nuxt/dist/index.mjs:1825:7)
  at async loadNuxt (/home/user/workspace/my-project/node_modules/nuxt/dist/index.mjs:1857:5)
  at async loadNuxt (/home/user/workspace/my-project/node_modules/@nuxt/kit/dist/index.mjs:493:19)
  at async Object.invoke (/home/user/workspace/my-project/node_modules/nuxi/dist/chunks/prepare.mjs:30:18)
  at async _main (/home/user/workspace/my-project/node_modules/nuxi/dist/cli.mjs:50:20)


```

**Comment by ErriourMe** [comment]
The same problem (from RC14 btw),

**Comment by AnoRebel** [comment]
I'm having the same issue with the stable Nuxt 3.0.0, any fixes?

**Comment by markus-gx** [comment]
Same error; Just commented to subscribe this topic :)

**Comment by ricardogobbosouza** [comment]
Nuxt3 Support in progress https://github.com/nuxt-community/eslint-module/pull/70

**Comment by vedmant** [comment]
Nuxt 3 team is doing really stupid releases, like 14 release candidates and on actual release bunch of breaking changes, is there any logic...

**Comment by mklueh** [comment]
@vedmant indeed a bit surprising

**Comment by ricardogobbosouza** [comment]
Released [v4.0.0](https://github.com/nuxt-community/eslint-module/releases/tag/v4.0.0)
